pkgs/tools/audio/google-music-scripts/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, python3, substituteAll, locale }:
+
+let
+  click6 = python3.pkgs.click.overridePythonAttrs (oldAttrs: rec {
```

Comment:
If older version of click is necessary, could you open an upstream bug asking for them to update to click 7?

Reply:
I realized that just by bumping the version number click 7 works fine, so I'll use a patch for that and submit it upstream.

Reply:
thebigmunch/google-music-scripts#21